### ISSUE TITLE
Installation instructions don't work on npm 1.x/2.x

### ISSUE THREAD

**Issue by markrian** [radify/karma-es6-shim#13]
The installation instructions say to simply run `npm install karma-es6-shim --save`. This works for npm3, but not for earlier versions, because they do not install packages as flatly as possible.

It seems the `lib/index.js` file makes the assumption that es(5|6)-shim will be installed in the top-level node_modules directory. See:

```
files.unshift(createPattern(__dirname + '/../../es6-shim/es6-sham.min.js'));
```

So here `__dirname` might be `~/my-awesome-app/node_modules/karma-es6-shim/lib`, which means the combined path string resolves to `~/my-awesome-app/node_modules/es6-shim/es6-sham.min.js`.

If the installation is done on npm3, that path is valid (and even then, it might not be, if the app itself depends on a different versions of es(5|6)-shim to what karma-es6-shim depends on.

Anyway - it seems like you should either revert back to specifying peerDependencies, or make use of [require.resolve()](https://nodejs.org/api/globals.html#globals_require_resolve), or something.

**Comment by gavD** [comment]
thanks for this. I'll take a look and see if I can figure out something that works for older NPM versions.

**Comment by gavD** [comment]
Hi,

I've finally gotten round to fixing this, thanks for your help :-) Version 1.0.0 of karma-es6-shim is compatible with older versions of NPM.

I have also released version [1.0.0 of the sample project](https://github.com/radify/karma-es6-shim-example).

Thanks,
- Gavin
